### PR TITLE
stencil swift kit filters support

### PR DESCRIPTION
**Pull request by svastven**
# Short description 📝

> StencilSwiftKit adds support for several other filters on top of Stencil. Although they are referenced in the [tuist scaffold docs](https://tuist.io/docs/commands/scaffold/#scaffold-command) their usage resulted in the following error

```
- Localized description: The operation couldn't be completed. (Stencil.TemplateSyntaxError error 1.)
- Error: Unknown filter 'upperFirstLetter'."
```

### Solution 📦

> The only thing that needed to be done was to fully import `StencilSwiftKit` into `TemplateGenerator.swift`

### Implementation 👩‍💻👨‍💻

import `StencilSwiftKit` to `TemplateGenerator.swift`

### CODE REVIEW

**Review by fortmarek — APPROVED** [review]
Changes look good to me, +1 for the acceptance test 🙂

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 
 ## Next
 
+- Add support for `StencilSwiftKit`'s additional filters. [#1994](https://github.com/tuist/tuist/pull/1994) by [@svastven](https://github.com/svastven).
```
Comment (laxmorek) [inline CHANGELOG.md:7]: It is not PR related but can you add `###Added` section's header?

**Review by laxmorek — APPROVED** [review]
🎉

File: CHANGELOG.md (hunk 1 above)
Comment (fortmarek) [reply CHANGELOG.md:7]: I've already added it in this [PR](https://github.com/tuist/tuist/pull/1993), so I think it's fine to keep it as-is, but good catch 👍 🙂

**Review by pepibumur — APPROVED** [review]

**Pull request merged**